src/cli/record/logOption.ts

```diff
@@ -0,0 +1,33 @@
+import type { Arguments, Options } from "yargs";
+import type { LogConfigLevel } from "../../utils/log";
+import { logger, SUPPORTED_LOG_CONFIG_LEVELS } from "../../utils/log";
+
+export const logOptions: { [key: string]: Options } = {
+  "log-level": {
+    describe: "Log config level",
+    default: "info",
+    choices: SUPPORTED_LOG_CONFIG_LEVELS,
+    requiresArg: true,
+  },
+  verbose: {
+    describe: "Verbose mode",
```

Comment:
It's better to describe what this option will do.

```suggestion
    describe: "Set log config level to \"debug\"",
```